doc/source/wsrep-status-index.rst

```diff
@@ -179,6 +179,10 @@ The sequence number of the current transaction in IST.
 
 The sequence number of the first transaction in IST.
 
+.. variable:: wsrep_last_applied
+
+Sequence number of the last applied transaction.
+
```

Comment:
please add a version info, as this variable isn't\wasn't available in all versions.